src/App.css

```diff
@@ -1,38 +1,347 @@
+  background-color: rgba(26, 83, 92, 0.9);
+  border: rgba(26, 83, 92, 0.9) 2px solid;
+  border-radius: 5px;
+  color: white;
+  font-size: 1.2rem;
+  text-decoration: none;  
+  padding: 0.5rem 1rem;
+  transition: color 0.3s ease-in-out, background-color 0.3s ease-in-out, border 0.3s ease-in-out;
+  -webkit-box-shadow: 8px 8px 0px 4px #1A535C; 
+  box-shadow: 8px 8px 0px 4px #1A535C;
+}
+
+.landingNav a:hover {
+  color: #1A535C;
+  background-color: white;
+  border: #1A535C 2px solid;
+}
+
+.landingNav a:focus {
+  color: #1A535C;   
+  background-color: white;
+}
+
+/* form styling */
+
+form {
+	margin: 1rem 0;
+	padding: 1rem 0 ;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+  border: #1A535C 2px solid;
+  border-radius: 5px;
+}
+
+form label {
+  font-weight: bolder;
+}
+
+form input {
+  margin: 0.25rem 0;
+  color: #000;
+  text-align: center;
+  width: 70%;
+  padding: 0.25rem 0;
+  border-radius: 5px;
+	text-align: center;
+
+}
+
+form button {
+  margin-top: 1rem;
+  background-color: #1A535C;
+  border: #1A535C 2px solid;
+  color: #ffffff;
+  border-radius: 5px;
+  padding: 0.25rem 0.5rem;
+  transition: 0.3s color ease-in-out, background-color 0.3s ease-in-out ;
+}
+
+form button:hover {
+  color: #1A535C;
+  background-color: white;
+}
+
+form button:focus {
+  color: #1A535C;
+  background-color: white;
 }
 
-.App-link {
-  color: #61dafb;
+form select {
+  width: 70%;
+  margin: 0.25rem 0;
+  background-color: white;
+  border: #1A535C 2px solid;
+  border-radius: 5px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+/* search results section styling */
+
+.results a {
+  transition: color 0.3s ease-in-out;
+}
+
+.results a:hover {
+  color: #ff6b6b;
+}
+
+.results a:focus {
+  color: #ff6b6b;
+}
+
+.searchParent .returnError {
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.searchParent .returnError {
+  text-align: center;
+  margin-bottom: 2rem;
+
+  
+/* animation for the error sentence to fade away  */
+  animation:errorFade 5s 1;
+  -webkit-animation:errorFade 5s 1;
+  animation-fill-mode: forwards;
+
+  animation-delay:2s;
+  -webkit-animation-delay:1s; /* Safari and Chrome */
+  -webkit-animation-fill-mode: forwards;
+}
+
+/* animation for the error sentence to fade away  */
+@keyframes errorFade {
+  from {opacity :1;}
+  to {opacity :0;}
+}
+
+.searchResult {
+  display: flex;
+  justify-content: center;
+}
+
+.searchResult ul {
+
+	padding: 1rem 0
+}
+
+.searchResult li {
+	padding-bottom: 0.25rem;
+}
+
+.searchResult li span {
+	text-transform: capitalize;
+	font-weight: bold;
+}
+
+/* chart styling */
+.wrapper .artistAlbums,
+.wrapper .chartResults {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: auto;
+  gap: 0.5rem;
+  margin-bottom: 2rem;
+}
+
+.artistAlbums .albumContainer,
+.chartResults .trackContainer {
+  border: #1A535C 2px solid;
+  border-radius: 5px;
+  padding: 1rem 0 0.75rem 0.5rem;
```

Comment:
Maybe add a touch more padding to the track container, as the text gets very close to the border on the right side when the song names are long!